### PR TITLE
chore: Updates Aries

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: cmd/agent-rest/go.mod [hunk 1]
```diff
@@ -1,27 +1,21 @@
-// Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (troyronda) [inline cmd/agent-rest/go.mod:1]: will need to put license header back.

File: go.mod [hunk 2]
```diff
@@ -1,36 +1,23 @@
-// Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (troyronda) [inline go.mod:1]: will need to put license header back.

File: cmd/agent-js-worker/go.mod [hunk 3]
```diff
@@ -11,11 +11,11 @@ require (
 	github.com/google/tink/go v1.5.0
 	github.com/google/uuid v1.2.0
 	github.com/hyperledger/aries-framework-go v0.1.6-0.20210304035610-cb1ce0da0c3b
```
Comment (sudeshrshetty) [inline cmd/agent-js-worker/go.mod:13]: aries version is wrong

**Review by sudeshrshetty — APPROVED** [review]

**Review by sudeshrshetty — CHANGES_REQUESTED** [review]
update agent JS & mobile

**Review by fqutishat — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**